selenoid_test.go

```diff
@@ -980,6 +981,75 @@ func TestAddedSeCdpCapability(t *testing.T) {
 	queue.Release()
 }
 
+func TestBidi(t *testing.T) {
```

Comment:
Check that bidi protocol works correctly (code used from devtools test with some changes)

Reply:
It is e2e test, right?